server/src/pages/homepage/view.js

```diff
@@ -9,14 +9,14 @@ class Head extends React.Component {
       <reactruntime.HeadTemplate {...this.props}>
         <script src={this.props.staticLink("/static/js/homepage-bundle.js")} async></script>
         <meta name="viewport" content="width=320, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" />
-        <meta name="description" content="Share anything on the web with anyone using Firefox Screenshots." />
+        <meta name="description" content="Share anything on the web with anyone using Firefox Screenshots." /> <!-- todo l10n: homePageDescription -->
```

Comment:
Ah, good question! Sorry that the README is lacking. Your questions are actually super helpful in making me understand what the docs should include. :)

There's an example app demonstrating attribute localization at https://github.com/projectfluent/fluent.js/tree/master/fluent-react/examples/text-input

If you wrap an element with `Localized` like this:

        <Localized id="type-name">
          <input
            type="text"
            placeholder="Your name"
            onChange={evt => this.handleNameChange(evt.target.value)}
            value={name}
          />
        </Localized>

You can define an attribute translation in FTL:

    type-name
        .placeholder = Your name

More on attributes: http://projectfluent.io/fluent/guide/attributes.html This should work to `meta` elements, too.